src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -359,12 +336,12 @@ def forward(
         output_attentions=False,
     ):
         # hidden_states: [batch_size, seq_length, hidden_size]
-        # repeat alibi tensor with the batch size
-        alibi = alibi.repeat(hidden_states.shape[0], 1, 1).to(hidden_states.device)
-
         # apply preprocessing if the input is padded
-        if attention_mask is not None and 0 in attention_mask:
```

Comment:
... and here as well :)

Reply:
This code snippet has been already erased ! It was in the previous version